frontend/src/views/omni/Modals/DownloadInstallationMedia.vue

```diff
@@ -311,7 +311,7 @@ const createSchematic = async () => {
       meta_values: {},
     };
 
-    if (labels.value) {
+    if (labels.value && Object.keys(labels.value).length > 0) {
```

Comment:
If no labels were set, we were setting here a YAML with empty content for the labels key. Now we do not set any META information if there are no labels.